__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -26,7 +27,8 @@ const savePrepReplacementExpectation = [
   'status',
   'createdAt',
   'updatedAt',
-  {}
+  {},
+  []
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `attachmentJson` field should be represented as a string containing serialized JSON.
```diff
-  {},
-  []
+  '{}',
+  '[]'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
It's correct because this is not string. String only when we saving to sqlite